common/script_values/babylon_values.txt

```diff
@@ -864,12 +888,93 @@ babylon_qing_fragile_unity_value = {
 		}
 		add = 5
 	}
+	# Each of these countries add 10 if they dont exists
+	if = {
+		limit = {
+			NOT = {
+				exists = c:XIN
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:MGL
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:MCH
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:XIB
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:TIB
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:SIH
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:HNA
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:YUE
+			}
+		}
+		add = 10
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:FRS
+			}
+		}
+		add = 5
+	}
+	if = {
+		limit = {
+			NOT = {
+				exists = c:KOR
+			}
+		}
+		add = 5
+	}
```

Comment:
Which led to this; if a country didnt exists it was free real estate for Qing fragile unity-wise, and since you cant scope through non-existent countries (to my knowledge) this happened..